docs/GettingStarted.md

```diff
@@ -952,6 +952,27 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | --- | --- | --- |
 | ``service_name`` | Service name used for `sidekiq` instrumentation | sidekiq |
 
+### DelayedJob
```

Comment:
We're missing important documentation; this section needs to be added to the list of links above, and the table of supported integrations before that, with appropriate details and links to resources wherever required.

Reply:
Looks like this has since been added; thanks!